Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1633,20 +1644,34 @@ RcTcacheEntry TextureCacheBase::CreateTextureEntry(
 #endif
 
   RcTcacheEntry entry;
-  if (custom_texture_data && custom_texture_data->m_levels.size() >= 1)
-  {
-    const u32 texLevels = no_mips ? 1 : (u32)custom_texture_data->m_levels.size();
-    const auto& first_level = custom_texture_data->m_levels[0];
-    const TextureConfig config(first_level.width, first_level.height, texLevels, 1, 1,
-                               first_level.format, 0);
+  if (!assets_data.empty() &&
+      std::none_of(assets_data.begin(), assets_data.end(), [](auto asset) { return !asset; }))
```

Comment:
`(const auto& asset)`. I'm also not sure if this condition is needed at all, this should never get empty shared_ptrs in assets_data, no?

Reply:
If you do want to check for this put it in an assert so we know this is a logic error.